python/lsst/afw/math/__init__.py

```diff
@@ -24,5 +24,6 @@
 """
 from __future__ import absolute_import
 from .mathLib import *
-from .warper import *
-from .background import *
+from .detail import *
+#from .warper import *
+#from .background import *
```

Comment:
These are commented because the packages imported in warper and background have not been wrapped yet, but they will eventually, so we will need to uncomment these lines.